transmart-core-api/src/main/groovy/org/transmartproject/core/users/MultidimQueryAuthorisationChecks.groovy

```diff
@@ -0,0 +1,25 @@
+package org.transmartproject.core.users
+
+import org.transmartproject.core.ontology.MDStudy
+
+/**
+ * 17.1 data model (study enitity, hypercube, constraints language) authorisation checks.
+ */
+interface MultidimQueryAuthorisationChecks {
```

Comment:
Please name this `AuthorisationChecks` to convey that this is the actual type people should use.

Reply:
sounds good. implemented.